ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/AlterTableHandler.java

```diff
@@ -135,8 +178,15 @@ private boolean handleForTableLevelReplication(Context withinContext) {
         return false;
       }
 
+      // In case of replace policy, even if the old table matches the new policy, none of the events including create
+      // table will be replayed as the old table is set of bootstrap by replace handler. So rename event can be skipped.
```

Comment:
There is nothing called replace handler. So, shall change the comment that "Old table is set for bootstrap due to replace policy."

Reply:
code is removed